modules/web/src/app/cluster/details/cluster/rbac/service-account/component.ts

```diff
@@ -0,0 +1,198 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, OnDestroy, OnInit, Input, Output, EventEmitter} from '@angular/core';
+import {ClusterServiceAccountService} from '@core/services/cluster-service-account';
+import {RBACService} from '@core/services/rbac';
+import {Subject, combineLatest} from 'rxjs';
+import {takeUntil, map, filter, switchMap, take} from 'rxjs/operators';
+import {MatTableDataSource} from '@angular/material/table';
+import {Cluster} from '@shared/entity/cluster';
+import {ClusterBinding, Binding, SimpleClusterBinding, Kind, ClusterServiceAccount} from '@shared/entity/rbac';
+import {MatDialog, MatDialogConfig} from '@angular/material/dialog';
+import {NotificationService} from '@core/services/notification';
+import {ConfirmationDialogComponent} from '@shared/components/confirmation-dialog/component';
+
+enum BindingMode {
+  Cluster = 'Cluster',
+  Namespace = 'Namespace',
+}
+
+enum Column {
+  StateArrow = 'stateArrow',
+  Name = 'name',
+  Namespace = 'namespace',
+  Actions = 'action',
+  Details = 'details',
+}
+
+@Component({
+  selector: 'km-rbac-service-account',
+  templateUrl: './template.html',
+})
+export class RBACServiceAccountComponent implements OnInit, OnDestroy {
+  private readonly _unsubscribe = new Subject<void>();
+  readonly Column = Column;
+  columns = [Column.StateArrow, Column.Name, Column.Namespace, Column.Actions];
+  isLoading = true;
+  dataSource = new MatTableDataSource<ClusterServiceAccount>();
+  bindingDetails: Record<string, SimpleClusterBinding[]>;
+  clusterServiceAccountExpansion: Record<string, boolean> = {};
+
+  @Input() cluster: Cluster;
+  @Input() projectID: string;
+  @Output() addBindingToServiceAccount = new EventEmitter<ClusterServiceAccount>();
+  @Output() deleteBinding = new EventEmitter<SimpleClusterBinding>();
+
+  constructor(
+    private readonly _rbacService: RBACService,
+    private readonly _matDialog: MatDialog,
+    private readonly _notificationService: NotificationService,
+    private readonly _clusterServiceAccountService: ClusterServiceAccountService
+  ) {}
+
+  ngOnInit(): void {
+    this._getServiceAccounts();
+    this._getBindings();
+  }
+
+  ngOnDestroy(): void {
+    this._unsubscribe.next();
+    this._unsubscribe.complete();
+  }
+
+  expandRow({id}: ClusterServiceAccount): void {
+    this.clusterServiceAccountExpansion[id] = !this.clusterServiceAccountExpansion[id];
+  }
+
+  addBinding(serviceAccount: ClusterServiceAccount): void {
+    this.addBindingToServiceAccount.emit(serviceAccount);
+  }
+
+  download(serviceAccount: ClusterServiceAccount): void {
+    window.open(
+      this._clusterServiceAccountService.getKubeconfigUrl(
+        this.projectID,
+        this.cluster.id,
+        serviceAccount.namespace,
+        serviceAccount.id
+      ),
+      '_blank'
+    );
+  }
+
+  deleteServiceAccount(element: ClusterServiceAccount): void {
```

Comment:
We're emitting event to parent for deleting binding so I think we should emit event for this as well and then delete service account in parent component for consistency. This way we also don't need to add `_updateBindings` method here which is duplicate of parent component's method. Wdyt?